src/App.tsx

```diff
@@ -15,71 +16,86 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  };
```

Comment:
state doesn't have typization (type / interface), pls add it before `class App`

Reply:
Thank's, I didn't see it👍